src/views/Conference.vue

```diff
@@ -1,36 +1,59 @@
 <template>
   <Page class="league-page-container">
+    <Loading v-show="loading" />
     <div
+      v-show="!loading"
       ref="jitsi"
       class="jitsi-container"
     />
   </Page>
 </template>
 <script>
+import BrowserWindowMessageConnection from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/connection/browser-window-message';
 import JitsiMeetExternalAPI from 'jitsi-iframe-api';
 import Page from '../components/layout/Page.vue';
 import { IS_MOBILE_DEVICE } from '../utils/util';
+import Loading from '../components/Loading.vue';
 
 export default {
   name: 'Conference',
   components: {
     Page,
+    Loading,
   },
   props: {
-    room: { type: String, required: true },
+    room: { type: String, default: '' },
   },
-  mounted() {
+  data: () => ({
+    connection: null,
+    loading: true,
+  }),
+  async mounted() {
+    this.connection = await BrowserWindowMessageConnection({
+      origin: `https://${process.env.VUE_APP_JITSI_URL}`,
+    });
+
+    this.connection.connect(({ room }) => {
+      this.$router.push({ name: 'conference', params: { room } });
+    });
+
```

Comment:
you may use `$once` to avoid putting `connection` to component's state:
```suggestion

    this.$once('hook:destroyed', () => connection.disconnect());
```

Reply:
Thanks. Not sure why but
```js
this.$once('hook:destroyed', () => {
      console.log('destroyed');
    });
```
and also 
```js
destroyed() {
    console.log('destroyed');
  },
```
does not log anything. the same with `beforeDestroy

Reply:
It is because of KeepAlive around RouterView (https://github.com/aeternity/superhero-ui/issues/664)

Reply:
Agree that `state management` makes app more predictable.
For this time I will use `deactivated hook` it works.

Reply:
deactivated hook makes that:
- we unsubscribed 
- next we back to the page
- there is no listener anymore

I will leave it with destroy and it will be subscribed till the keep alive.

Reply:
Docs says that `max` is `The maximum number of component instances to cache.` not sure when unsubscribed will be called. But at least after we open the page second time it use cache and don't subscribe again.